pr-scripts/create_pr.sh

```diff
@@ -26,12 +26,20 @@ FILEPATH="$4"
 if [ $REPO = "eks-distro-build-tooling" ]; then
     CHANGED_FILE="Tag file"
 elif [ $REPO = "eks-distro" ]; then
-    CHANGED_FILE="Makefile"
+    CHANGED_FILE="Makefiles"
+elif [ $REPO = "eks-distro-prow-jobs" ]; then
+    CHANGED_FILE="Prow jobs"
 fi
 
+COMMIT_MESSAGE="Update EKS Distro base image tag"
```

Comment:
Let's make it clear that the PR is coming from a bot, like `[PR Bot] ...`?

Reply:
Sounds good!